sonic_platform_base/sonic_sfp/sfputilhelper.py

```diff
@@ -41,7 +44,7 @@ class SfpUtilHelper(object):
     def __init__(self):
         pass
 
-    def read_porttab_mappings(self, porttabfile):
+    def read_porttab_mappings(self, porttabfile, asic_inst = 0):
```

Comment:
Remove spaces around equals sign for default parameter assignments.

Reply:
Looks like this was missed in the most recent commit.

Reply:
Thanks Joe, I have updated the space here as well.